tests/test_bgp_neighbor_af.rb

```diff
@@ -267,6 +356,12 @@ def test_advertise_map
   def advertise_map(af, dbg)
     %w(advertise_map_exist advertise_map_non_exist).each do |k|
       v = %w(foo bar)
+      if platform == :ios_xr
+        assert_raises(ArgumentError) do
```

Comment:
As above, I would really expect the default behavior (getter and default return `nil` and setter raises an `UnsupportedError`) rather than this ArgumentError.